common/time.py

```diff
@@ -1,3 +1,6 @@
 import datetime
 
 MIN_DATE = datetime.datetime(year=2023, month=6, day=1)
+
+def valid_datetime(d: datetime.datetime):
+  return d > MIN_DATE
```

Comment:
this helper is about the same length as "d > MIN_DATE" but lacks the locality of that. if you want a helper, think it would be better to make one that just checks if the current time is valid